collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -71,19 +71,12 @@ public static String getMimeType(String fileUrl) throws IOException {
     }
 
     public static boolean createFolder(String path) {
-        boolean made = true;
         File dir = new File(path);
-        if (!dir.exists()) {
-            made = dir.mkdirs();
-        }
-        return made;
+        return dir.exists() || dir.mkdirs();
     }
 
     public static byte[] getFileAsBytes(File file) {
-        byte[] bytes = null;
-        InputStream is = null;
-        try {
-            is = new FileInputStream(file);
+        try (InputStream is = new FileInputStream(file)) {
```

Comment:
Nice cleanup

Reply:
It was an unrelated but harmless change, so I went ahead and did it.